Apps/W1/Email - Outlook REST API/app/src/EmailOAuthClient.Codeunit.al

```diff
@@ -133,7 +140,7 @@ codeunit 4507 "Email - OAuth Client" implements "Email - OAuth Client"
         AccessToken: Text;
         TokenCache: Text;
     begin
-        if not OAuth2.AcquireOnBehalfAccessTokenAndTokenCache(OAuthAuthorityUrlTxt, '', GraphResourceURLTxt, AccessToken, TokenCache) then begin
+        if not OAuth2.AcquireOnBehalfAccessTokenAndTokenCache(StrSubstNo(OAuthAuthorityUrlTxt, TenantId), '', GraphResourceURLTxt, AccessToken, TokenCache) then begin
```

Comment:
Is there a reason you want to modify the SaaS logic path as well (is it to make sure guest / external users are getting the correct endpoint)?

Reply:
This change is needed because I changed the default value of OAuthAuthorityUrlTxt, inserting the "%1" instead of the "common"

Reply:
👍